### PR TITLE
Make arguments for python api key only

### PR DESCRIPTION
**Pull request by chraibi**

### CODE REVIEW

File: python_bindings_jupedsim/bindings_jupedsim.cpp [hunk 1]
```diff
@@ -155,8 +156,9 @@ PYBIND11_MODULE(py_jupedsim, m)
                double radius) {
                 JPS_VelocityModelBuilder_AddParameterProfile(w.handle, id, t, tau, v0, radius);
             },
+            py::kw_only(),
             py::arg("id"),
-            py::arg("t"),
+            py::arg("t_gap"),
```
Comment (Ozaq) [inline python_bindings_jupedsim/bindings_jupedsim.cpp:161]: please name it `time_gap` as you originally suggested

File: python_bindings_jupedsim/bindings_jupedsim.cpp [hunk 2]
```diff
@@ -338,6 +342,7 @@ PYBIND11_MODULE(py_jupedsim, m)
             py::arg("model"),
             py::arg("geometry"),
             py::arg("areas"),
+            py::kw_only(),
```
Comment (Ozaq) [inline python_bindings_jupedsim/bindings_jupedsim.cpp:345]: I would like all parameters on simulation to require named arguments

**Review by Ozaq — CHANGES_REQUESTED** [review]
Please also mandate named parameters on:
* `add_area`
* `switch_agent_profile`

File: python_bindings_jupedsim/bindings_jupedsim.cpp (hunk 2 above)
Comment (chraibi) [reply python_bindings_jupedsim/bindings_jupedsim.cpp]: Addressed both comments in one commit

**Review by Ozaq — APPROVED** [review]

**Pull request merged**